client/src/Movies/MovieCard.js

```diff
@@ -1,7 +1,25 @@
 import React from 'react';
 
 const MovieCard = props => {
```

Comment:
you an also destructure here: `const MovieCard = { title, director, metascore, stars } => (`